LICENSE-binary

```diff
@@ -930,7 +930,7 @@ The license texts of these dependencies can be found in the licenses directory.
         * JAX-RS provider for JSON content type (org.codehaus.jackson:jackson-jaxrs:1.9.13 - http://jackson.codehaus.org)
         * Xml Compatibility extensions for Jackson (org.codehaus.jackson:jackson-xc:1.9.13 - http://jackson.codehaus.org)
         * Javassist (org.javassist:javassist:3.24.1-GA - http://www.javassist.org/)
-        * RocksDB JNI (org.rocksdb:rocksdbjni:5.18.4 - http://rocksdb.org/)
+        * RocksDB JNI (org.rocksdb:rocksdbjni:6.27.3 - http://rocksdb.org/)
```

Comment:
same issue here I think:
As per the build error, this line needs to be changed to remove the trailing slash:
* RocksDB JNI (org.rocksdb:rocksdbjni:6.27.3 - http://rocksdb.org/)
should be 
* RocksDB JNI (org.rocksdb:rocksdbjni:6.27.3 - http://rocksdb.org)